htdocs/xoops_lib/vendor/smarty/smarty/demo/plugins/cacheresource.mysql.php

```
@@ -54,7 +54,7 @@ public function __construct()
         try {
             $this->db = new PDO("mysql:dbname=test;host=127.0.0.1", "smarty");
         } catch (PDOException $e) {
-            throw new SmartyException('Mysql Resource failed: ' . $e->getMessage());
+            throw new SmartyException('MySQL Resource failed: ' . $e->getMessage());
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '57', column '23'). (undefined)

(MissingImport)